src/robusta/core/model/runner_config.py

```diff
@@ -81,6 +82,20 @@ class RunnerConfig(BaseModel):
     active_playbooks: Optional[List[PlaybookDefinition]] = []
     alert_relabel: Optional[List[AlertRelabel]] = []
 
+    @root_validator()
+    def ensure_unique_sink_name(cls, val: Dict) -> Dict:
+        print(f"validating {val}")
```

Comment:
@nherment I think you forgot to remove print

Reply:
Thank you. :facepalm: